packages/desktop/views/dashboard/governance/views/DetailsView.svelte

```diff
@@ -73,10 +77,14 @@
         if (selectedProposalOverview) {
             const trackedParticipations = Object.values(selectedProposalOverview)
             const votes = calculateWeightedVotes(trackedParticipations)
-
-            votedAnswerValues =
-                trackedParticipations.find((overview) => overview.endMilestoneIndex === 0)?.answers ?? []
-            totalVotes = votes?.reduce((accumulator, votes) => accumulator + votes, 0) ?? 0
+            const lastActiveOverview = trackedParticipations.find((overview) => overview.endMilestoneIndex === 0)
+            const votesSum = votes?.reduce((accumulator, votes) => accumulator + votes, 0) ?? 0
+
+            votedAnswerValues = lastActiveOverview?.answers ?? []
+            totalVotes =
+                proposalState.status === ProposalStatus.Commencing
+                    ? parseInt(lastActiveOverview.amount, 10) ?? 0
```

Comment:
I receive an error if there hasn't been an overview.
```suggestion
                    ? parseInt(lastActiveOverview?.amount, 10) ?? 0
```